opentreemap/treemap/search.py

```diff
@@ -266,9 +262,12 @@ def fn(predicate_value, field=None):
             if isinstance(value, datetime):
                 date_value = value.date().isoformat()
                 inner_value = {field: date_value}
+            elif isinstance(value, float):
+                inner_value = {field: value}
             else:
-                raise ParseException("Cannot perform min/max comparisons on "
-                                     "non-date hstore fields at this time.")
+                raise ParseException(
+                    "Cannot perform min/max comparisons on "
+                    "non-date/numeric hstore fields at this time.")
```

Comment:
Should this now be "Cannot perform min/max comparisons on integer values at this time."?